external/harvestPackages/harvestPackages.depproj

```diff
@@ -29,6 +29,13 @@
         PackageIndexes="$(PackageIndexFile)">
         <Output  TaskParameter="LastStablePackages" ItemName="_NewPackageReferences" />
     </GetLastStablePackage>
+    
+    <ItemGroup>
+      <_NewPackageReferences Condition="'%(Identity)' == 'System.Reflection.TypeExtensions'">
+        <!-- harvest from 4.4.0 until we ship a new stable package -->
```

Comment:
Just wondering, why do you need this if the harvest depproj is already referencing this version and the pkgproj is specifying 4.4.0 as the harvestPackageVersion?

Reply:
It is to help catch us if/when we move past 4.4.0 so we update this project to stop harvesting from here.